libnmstate/state.py

```diff
@@ -132,6 +188,20 @@ def verify_interfaces(self, other_state):
 
         self._assert_interfaces_equal(other_state)
 
+    def verify_routes(self, other_state):
+        self._clean_routes()
```

Comment:
This could use the same docstring as the verify method for interfaces to clarify what it does. Actually I think it should be named to something like `is_subset()` since this is what it checks.

Reply:
`verify_routes()` is not checking `is_subset`, they should be identical.